core/src/io/anuke/mindustry/Validate.java

```diff
@@ -0,0 +1,21 @@
+package io.anuke.mindustry;
+
+/**
+ * Utility methods for parameter and expression validation.  API copied from Apache Lang Validate
+ * https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/Validate.html
+ */
+public class Validate{
+    /**
+     * Returns value or throws NPE if it is null
+     * @param value the variable to validate
+     * @param name  the variable name
+     * @param <T>   the type of value
+     * @return      value
+     */
+    public static <T> T notNull(T value, String name){
```

Comment:
I am against adding an entire new class for a single method that could be inlined. `Validate.notNull(x, "abc")` isn't much shorter than  `if(x == null) throw new NullPointerException("abc")`. It's also worth mentioning that utilities like this should not be added into the main package, which is for core things like Vars only.

Reply:
Well, if you look at the Apache Commons Lang Validate class, you can see that there are quite a number of useful validations.  I just didn't want to add them until I used them.  But having a helper method is much nicer because you can inline the validation, which you can't do with the throw statement.  I could add the helper method to the class where I used it, but the idea is to use it more broadly to help stop NPEs.  So having it in its own class makes more sense.  What do you think?